app/amp_start.py

```diff
@@ -22,7 +22,7 @@ class Handler(start.Handler):
 
     def get(self):
         self.env.amp = True
-        """The reason for setting params.lang instead of env.lang is to prevent 
-        the default lang=en from being set on the search index."""
+        # The reason for setting params.lang instead of env.lang is to prevent 
+        # the default lang=en from being set on the search index.
```

Comment:
Oops I should have also caught this together, but the default language is not always English (it's up to each repository). So it would be more precise to say something like:
> ... is not to attach lang parameter to the URL if it is not explicitly specified in the current request.

Reply:
Oh, I didn't notice that repository has own default language settings? Let's rewrite it to your suggesting comments.

Reply:
Yes, it's not very obvious, but the first language in the repository setting becomes the default language.